contractsV5/IKyberHint.sol

```diff
@@ -12,22 +12,22 @@ interface IKyberHint {
     }
 
     function parseEthToTokenHint(bytes calldata hint) external view
-        returns(HintType hintType, address[] memory reserves, uint[] memory splits, uint failureHint);
+    returns(HintType hintType, address[] memory reserves, uint[] memory splits, uint failureHint);
```

Comment:
Can we keep the same indentation standard among the interfaces? Compare the current IKyberHint and IKyberNetwork files and lemme know which to follow, thanks.

Reply:
I think per function the return value should be indented.